test/queryTest.js

```diff
@@ -56,7 +61,10 @@ describe('Query', function () {
                                 test.must(err).be.equal(null);
                                 columns[4].insert(timestampPoints, function (err) {
                                     test.must(err).be.equal(null);
-                                    done();
+                                    columns[5].insert(symbolPoints, function (err) {
+                                        test.must(err).be.equal(null);
+                                        done();
+                                    });
```

Comment:
Oh boy, isn't there another way instead of nesting??
Like: https://stackoverflow.com/questions/48663080/wait-for-all-different-promise-to-finish-nodejs-async-await

Reply:
Maybe, but IIRC the exposed function needs to be changed to have promises or async functions